roles/openshift_logging_fluentd/templates/fluentd.j2

```diff
@@ -120,6 +120,52 @@ spec:
         - name: "MUX_CLIENT_MODE"
           value: "{{ openshift_logging_mux_client_mode }}"
 {% endif %}
+
+{% if openshift_logging_fluentd_remote_syslog is defined %}
```

Comment:
```
 +{% if openshift_logging_fluentd_remote_syslog is defined and openshift_logging_fluentd_remote_syslog %} 
```
I think that's how you say "if openshift_logging_fluentd_remote_syslog is defined and openshift_logging_fluentd_remote_syslog is True"